app/models/mixins/assignment_mixin.rb

```diff
@@ -3,6 +3,17 @@
 module AssignmentMixin
   extend ActiveSupport::Concern
   ESCAPED_PREFIX = "escaped".freeze
+  NAMESPACE_SUFFIX = "assigned_to".freeze
+
+  # This module method was not intended to be mixed into a class. It is used outside
+  # of any class that may mix this in, (Classification) to find the assignments associated
+  # with a given tag.
+  def self.all_assignments(tag = nil)
+    scope = Tag.where(["name LIKE ?", "%/#{AssignmentMixin::NAMESPACE_SUFFIX}/%"])
+    scope = scope.where(["name LIKE ?", "%#{tag}"]) if tag.present?
```

Comment:
I feel like if there's a bug, it would be in the string matching.  As long as we have good tests, it looks ok.